src/auto.ts

```diff
@@ -492,9 +495,39 @@ export default class Auto {
     this.logger.verbose.info("Using command: 'shipit'");
     this.hooks.beforeShipIt.call();
 
+    const isPR = 'isPr' in env && env.isPr;
+    // env-ci sets branch to target branch (ex: master) in some CI services.
+    // so we should make sure we aren't in a PR just to be safe
+    const isMaster = !isPR && 'branch' in env && env.branch === 'master';
+    const publishInfo = isPR
+      ? await this.canary(options)
+      : isMaster
+      ? await this.publishLatest(options)
+      : undefined;
+
+    if (!isPR && !isMaster) {
+      this.logger.log.info(
+        'No version published. Could not detect if in PR or on master branch.'
```

Comment:
Just update the wording here to be `base branch` or use the `BASE_BRANCH` const mentioned above.

Reply:
https://github.com/intuit/auto/pull/357 constructs the string with whatever the base branch is. Do you think it would be better to just say base branch?